REFERENCE.md

```diff
@@ -6693,6 +6694,14 @@ Data type: `Any`
 
 Default value: ``undef``
 
+##### <a name="userdir"></a>`userdir`
+
+Data type: `Any`
+
+Directory out of which per-user content is loaded. It may be any valid `UserDir` directive values.
+
+Default value: The value of `$path`
```

Comment:
This looks manually written since it doesn't match what's in the `@param`. You can generate it using `rake strings:generate:reference`